### PR TITLE
Fix a few issues including max txn size

### PR DESCRIPTION
**Pull request by davidiw**
* Max txn size was in Move a a debug assert as well in our code base, so I migrated to just our code base
* I made it 256
* I removed json rpc config, since it has long since been dead
* Removed key manager config, which is also really dead

### CODE REVIEW

File: aptos-move/aptos-vm/src/aptos_vm_impl.rs [hunk 1]
```diff
@@ -196,8 +196,18 @@ impl AptosVMImpl {
         // The submitted transactions max gas units needs to be at least enough to cover the
         // intrinsic cost of the transaction as calculated against the size of the
         // underlying `RawTransaction`
-        let min_txn_fee =
-            gas_constants.to_external_units(calculate_intrinsic_gas(raw_bytes_len, gas_constants));
+        let min_txn_fee = {
+            let min_transaction_fee = gas_constants.min_transaction_gas_units;
```
Comment (zekun000) [inline aptos-move/aptos-vm/src/aptos_vm_impl.rs:200]: probably still want to have a function and explains why not use move one?

**Review by zekun000 — APPROVED** [review]

File: aptos-move/aptos-vm/src/aptos_vm_impl.rs (hunk 1 above)
Comment (davidiw) [reply aptos-move/aptos-vm/src/aptos_vm_impl.rs:200]: Better than magic numbers

File: aptos-move/framework/aptos-framework/sources/configs/VMConfig.move [hunk 2]
```diff
@@ -96,7 +96,7 @@ module AptosFramework::VMConfig {
             maximum_number_of_gas_units: 4000000,
             min_price_per_gas_unit,
             max_price_per_gas_unit: 10000,
-            max_transaction_size_in_bytes: 4096,
```
Comment (msmouse) [inline aptos-move/framework/aptos-framework/sources/configs/VMConfig.move:99]: ❤️

**Pull request closed without merging**